### ISSUE TITLE
[chart] current price (dotted line) shows incorrect value

### ISSUE THREAD

**Issue by tatyanakurinskaya** [gnosis/ido-ux#619]
1. open app
2. open some active auction
3. enter amount=2, price=20
4. look at current price (dotted line) in chart

Result: 
![Screen Shot 2021-05-26 at 09 38 49](https://user-images.githubusercontent.com/863519/119613868-46182200-be06-11eb-9503-468d72c2b804.png)

Exp.result: it should point to 2 USDC per DAI

**Comment by ramirotw** [comment]
@josojo is the update in the orders book chart clearing price intentional whenever you modify the place order form or should it stay fixed until the user places the order?

**Comment by ramirotw** [comment]
> I see that it is confusing that the graph is updating, but the auction details is not updating, though I think we are good with the current ticket from henry #629.
> 
> Wdyt?

Agreed. If this gets reported by more users we could rethink the UX so it's clear that the graph is updated when interacting with the form.
